src/firefly/java/edu/caltech/ipac/firefly/server/query/lsst/LSSTQuery.java

```diff
@@ -36,11 +36,13 @@
  */
 public abstract class LSSTQuery extends IpacTablePartProcessor {
     private static final Logger.LoggerImpl _log = Logger.getLogger();
-    public static final String PORT = AppProperties.getProperty("lsst.dax.port","5001");
-    public static final String HOST = AppProperties.getProperty("lsst.dax.hostname","lsst-qserv-dax01.ncsa.illinois.edu");
+    public static final String HOST = AppProperties.getProperty("lsst.dax.hostname","https://lsst-pdac.ncsa.illinois.edu");
```

Comment:
I just noticed that this is one of the SearchProcessors that has not been converted to EmbeddedDbProcessor and is writing results into ipac table first.  I must have missed it the last time I did the conversion.  This may be a good time to do so.